packages/flame/lib/src/game/game.dart

```diff
@@ -216,6 +216,13 @@ abstract class Game {
   /// - GameWidget
   /// - [Game.overlays]
   final overlays = ActiveOverlaysNotifier();
+
+  /// Use this method in case you need to project a coordinate received by a
+  /// gesture/pointer event to your game coordinate system.
+  ///
+  /// By default this just returns the same received coordinate transformed to
+  /// a Vector2, override this to add addtional logic to that projection
+  Vector2 projectCoordinate(Offset value) => value.toVector2();
```

Comment:
`projectCoordinates`, it would only be singular in a one-dimensional coordinate system